pkg/sanity/controller.go

```diff
@@ -48,6 +48,11 @@ func verifyVolumeInfo(v *csi.Volume) {
 	Expect(v.GetId()).NotTo(BeEmpty())
 }
 
+func verifySnapshotInfo(snapshot *csi.Snapshot) {
+	Expect(snapshot).NotTo(BeNil())
+	Expect(snapshot.GetId()).NotTo(BeEmpty())
```

Comment:
I think we can add some more verification here.

`SourceVolumeID` is REQUIRED.

Snapshot definition talks about `SizeBytes`.
> The size of the volume MUST NOT  be less than the size of the source snapshot.

Let's add those.

Reply:
Updated for check attributes  that is REQUIRED. As for `SizeBytes`, 
>The purpose of this field is to give CO guidance on how much space is needed to
	// create a volume from this snapshot. The size of the volume MUST NOT
	// be less than the size of the source snapshot.

I think it is used to CO when restore volume from a snapshot to verify there is enough space. So we do not need verify here.